core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -337,6 +337,9 @@ object Defaults {
   val LiLeaderElectionOnCorruptionWaitMs = 0L
   val LiZookeeperPaginationEnable = false
   val LiRackIdMapperClassNameForRackAwareReplicaAssignment: String = null
+  // Waiting for kicking out a dead host from ISR is common.  It's a good starting point to use 2 * Defaults.ReplicaLagTimeMaxMs.
+  val LiLongTailProduceRequestLogThresholdMs = 2 * Defaults.ReplicaLagTimeMaxMs
```

Comment:
One full minute? I guess it's fine as a default (to keep logging near zero), but it's much bigger than I'd expect we'd want for Espresso.

Reply:
This one is intended as a default value (to log almost nothing).
Internally, this should be configured to something else (e.g. p999, or lower in krc per espresso asks)